cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/query_enhancements/04/queries_ui.spec.js

```diff
@@ -116,7 +116,8 @@ export const runQueryTests = () => {
               const version = versionText.replace('v ', '').trim();
               let docsVersion;
 
-              if (version === '3.0.0') {
+              // FIXME: using version to decide docsVersion is fragile, each time the version bumped, the test needs to updated accordingly
+              if (version === '3.1.0') {
```

Comment:
nit: we can further enhancement this by reading the version from package.json

Reply:
I'm prefer to read branch from package.json here to align with the implementation in [docs_link_service](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/core/public/doc_links/doc_links_service.ts#L52).